versioned/storage/common/src/main/java/org/projectnessie/versioned/storage/common/persist/ObjTypes.java

```diff
@@ -55,11 +60,23 @@ public static ObjType forShortName(@Nonnull String shortName) {
   private static final class Registry {
     private static final Map<String, ObjType> BY_ANY_NAME;
     private static final Set<ObjType> OBJ_TYPES;
+    private static final List<ObjTypeBundle.ObjTypeMapper> OBJ_TYPE_MAPPERS;
+
+    static ObjType maybeMapped(String name) {
+      for (ObjTypeBundle.ObjTypeMapper mapper : OBJ_TYPE_MAPPERS) {
+        ObjType type = mapper.mapGenericObjType(name);
+        if (type != null) {
```

Comment:
Is it even possible that `type` could be null here? Since `GenericObjTypeBundle` never returns null, and is always registered.

Reply:
With the added commit, nope.

The original plan was that there can be multiple fallbacks, which could decide whether they support it or not. But processing order would likely make that non-deterministic.